src/ld65/config.c

```diff
@@ -757,6 +758,7 @@ static void ParseSegments (void)
                         case CFGTOK_RW:    /* Default */                    break;
                         case CFGTOK_BSS:   S->Flags |= SF_BSS;              break;
                         case CFGTOK_ZP:    S->Flags |= (SF_BSS | SF_ZP);    break;
+                        case CFGTOK_OVERLAY: S->Flags |= SF_OVERLAY;        break;
```

Comment:
The constant needs to be `(SF_RO | SF_REPLACE)`.  It will avoid an error message if the memory area is marked as Read-Only.